ndcube/visualization/mpl_sequence_plotter.py

```diff
@@ -113,7 +113,7 @@ def __init__(self, sequence, sequence_axis_coords=None, sequence_axis_unit=None,
 
     def _sequence_slider_function(self, val, artist, slider):
         self._sequence_idx = int(val)
-        self.data, self.wcs, _plot_axes, _coord_params = self._cubes[self._sequence_idx].plotter._prep_animate_args(
+        self.data, self.wcs, _, _ = self._cubes[self._sequence_idx].plotter._prep_animate_args(
```

Comment:
What does this change do exactly?

Reply:
It just highlights the fact that the last two outputs are unused.